featuretools/synthesis/deep_feature_synthesis.py

```diff
@@ -242,7 +246,7 @@ def __init__(self,
                                            self.ignore_entities,
                                            self.ignore_variables,
                                            self.es)
-        self.seed_features = seed_features or []
+        self.seed_features = sorted(seed_features or [], key=lambda f: f.primitive.name)
```

Comment:
why not sort by unique name, for features?

Reply:
Good point--changed